user-console/src/main/resources/org/pentaho/mantle/public/browser/js/browser.templates.js

```diff
@@ -113,19 +113,19 @@ define([
 
   //file template
   templates.file = Handlebars.compile(
-      "{{#ifCond folder 'false' }}" +
+      "{{#ifCond folder false}}" +
           "{{#if trash}}" +
-          "<div id='{{id}}' class='file' origPath='{{origPath}}' path='{{path}}' type='file' ext='{{trashPath}}' title='{{trashPath}}' role='option'>" +
+          "<div id='{{objectId}}' class='file' origPath='{{origPath}}' path='{{path}}' type='file' ext='{{trashPath}}' title='{{trashPath}}' role='option'>" +
```

Comment:
The `origPath` does not seem to be being used and could be removed. Especially because the new REST service does not have that field.

Reply:
`origPath` is a concept tied to the trash bin for restoring file's to their original paths. It only shows up in the DOM when there is something in the trash. Here is an example where I created a folder `asdfasdfasdf` in home/admin, and then moved it to the trash:

```<div id="1056c9f3-e84c-4624-bf8a-60176b84f808" class="file selected" origpath="Origin: /home/admin/asdfasdfsadf" path="/home/admin/.trash/pho:1056c9f3-e84c-4624-bf8a-60176b84f808/asdfasdfsadf" type="folder" ext="Origin: /home/admin/asdfasdfsadf" title="Origin: /home/admin/asdfasdfsadf" role="option" aria-selected="true"><div class="icon trashFolder"> </div><div class="title">asdfasdfsadf</div></div>

Reply:
I could not find any usages of this attribute. It's being put there, but not used, afaict.

Reply:
ok, good to know, I will get to it as I review all Folder/File actions

Reply:
I believe this is the last comment... I will look into this now :D

Reply:
While I agree that it would be nice to remove unused fields, I cannot prove that this property is _not_ used anywhere...

I [searched for references](https://github.com/search?q=org%3Apentaho%20origPath&type=code) to the attribute and it seemed sparse...

I also looked at the history and saw that the attribute was introduced [here](https://github.com/pentaho/pentaho-platform/commit/d70c64fd93ac9d504bfa8a6d9a098edd003f1192)

I don't see any additional references to it in the commit and I cannot find the original issue on Jira (probably because it is so old): `BISERVER-9528`

Like some of the other "nice to have" changes here, I don't feel confident removing it now without understanding it more... I've already changed a lot with this work, I don't want to introduce a regression related to something a decade old. I'd prefer to let this all "settle" (test and integrate etc.) and then come back to this... thoughts?

Reply:
Sure, that's fine. Let it be.